ludwig/features/base_feature.py

```diff
@@ -307,6 +307,8 @@ def concat_dependencies_and_build_output(
         final_hidden[self.name] = (feature_hidden, feature_hidden_size)
 
         # ================ Outputs ================
+        kwargs['is_training'] = is_training
+        kwargs['dropout_rate'] = dropout_rate
```

Comment:
this is suspicious, need to investigate, why do you need it?

Reply:
What's happening is most features don't require the flag is_training in the build_output method. But if you use a Dense Decoder, you need that flag. concat_dependencies_and_build_output is passing the kwargs but it's not passing is_training and dropout_rate to build_output. 

Since build_output needs to have the same signature for all the features, is_training and dropout_rate need to be in the kwargs. I didn't see any way around it.

Reply:
Got it, then the cleaner solution is to add those two parameters to self.build_output

Reply:
yes, It will require changes in a few more places but I can do that.